zebrad/src/components/mempool/tests.rs

```diff
@@ -406,23 +406,6 @@ async fn mempool_cancel_mined() -> Result<(), Report> {
         .await
         .unwrap();
 
-    // Queue transaction from block 2 for download
-    let txid = block2.transactions[0].unmined_id();
-    let response = mempool
-        .ready_and()
-        .await
-        .unwrap()
-        .call(Request::Queue(vec![txid.into()]))
-        .await
-        .unwrap();
-    let queued_responses = match response {
-        Response::Queued(queue_responses) => queue_responses,
-        _ => unreachable!("will never happen in this test"),
-    };
-    assert_eq!(queued_responses.len(), 1);
-    assert!(queued_responses[0].is_ok());
-    assert_eq!(mempool.tx_downloads().in_flight(), 1);
```

Comment:
This was just moved below. The test would pass but for the wrong reason (the download was cancelled due to the network upgrade and not because it was mined)